www/install/php/Update-20.10.0.post.php

```diff
@@ -31,62 +31,70 @@
 try {
     $pearDB->beginTransaction();
     /**
-     * 'platform_status' feature
+     * register server to 'platform_status' table
      */
-    // Insert the central as first platform and parent of all others
-    $errorMessage = "Unable to insert the central in the platform_topology table.";
-    $centralServerQuery = $pearDB->query("SELECT `id`, `name` FROM nagios_server WHERE localhost = '1'");
-    if ($row = $centralServerQuery->fetch()) {
+    // Correct 'isCentral' flag value
+    $errorMessage = "Unable to get server data from the 'informations' table.";
```

Comment:
Usually, we do not give information about the database structure in error messages to the customer.